.ci/.jenkins-schema.yml

```
@@ -0,0 +1,16 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
+    SPEC_FILEPATH: "test/Elastic.Apm.Tests/TestResources/json-spec"
+  - REPO: "apm-agent-go"
+    SPEC_FILEPATH: "internal/apmschema/jsonschema"
+  - REPO: "apm-agent-java"
+    SPEC_FILEPATH: "apm-agent-core/src/test/resources/apm-server-schema/current"
+  - REPO: "apm-agent-nodejs"
+    SPEC_FILEPATH: "test/integration/api-schema/apm-server-schema"
+  - REPO: "apm-agent-php"
```

@SergeyKleyman pointed me to use https://github.com/elastic/apm-agent-php/tree/master/tests/APM_Server_intake_API_schema/latest_used

```suggestion
  - REPO: "apm-agent-php"
    SPEC_FILEPATH: "tests/APM_Server_intake_API_schema/latest_used"
```